src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -271,13 +272,38 @@ def run_job(self, job: LandingJob) -> bool:
 
             # Run through the patches one by one and try to apply them.
             for revision in job.revisions.all():
-                patch_buf = StringIO(revision.patch_string)
+                # TODO: Rather than parsing the patch details from the full HG patch
+                # stored in the job, we should read the revision's metadata (and
+                # move to only store the diff in the patch_string, rather than an
+                # export).
```

Comment:
Could you file a bug for this and reference it here?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1936171